intermediate_source/dynamic_quantization_bert_tutorial.rst

```diff
@@ -523,13 +523,33 @@ processing the evaluation of MRPC dataset.
 We can serialize and save the quantized model for the future use using
 `torch.jit.save` after tracing the model.
 
+.. code:: python
+
+    import random 
+    global_rng = random.Random()
+
+    def ids_tensor(shape, vocab_size, rng=None, name=None):
```

Comment:
I don't see the `name=None` parameter being used anywhere . . . Is that necessary or can be delete for brevity?

Reply:
Yeah that is true, let me check that once!

Reply:
Can you please check the changes once! Also there was an other issue #1406 for the same tutorial in which the colab file needs changes. Can you point me out on how to get the colab edit access or do I need to make a new colab file from scratch?